packages/react/src/VideoPlayer/components/VideoTooltip/VideoTooltip.tsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react'
+import clsx from 'clsx'
+
+import {Text} from '../../../Text'
+import styles from '../../VideoPlayer.module.css'
+
+type VideoTooltipProps = React.HTMLAttributes<HTMLDivElement>
+
+export const VideoTooltip = ({children, className, ...rest}: VideoTooltipProps) => (
```

Comment:
Is it possible to use our [Tooltip](https://primer-2c317f85db-26139705.drafts.github.io/storybook/?path=/story/components-tooltip--default) component here instead of maintaining a custom one? It uses popover.

Reply:
Great suggestion, thanks. I'm looking into this now

Reply:
This works, but there's a bit of weirdness with the popover positioning. I'm not 100% sure yet, but I think we're fighting over the popover's position. The range input is trying to position it by the cursor, and the popover is trying to position itself centrally. 

Would you be happy for this to go in as part of a subsequent PR?

Reply:
(A video of a video player is slightly confusing 😅)

Reply:
> Would you be happy for this to go in as part of a subsequent PR?

Yeah, sounds good 👍